### ISSUE TITLE
Rails/TimeZone recommends `DateTime.zone.now`, which doesn't exist

### ISSUE THREAD

**Issue by maxjacobson** [bbatsov/rubocop#2151]
```
>> DateTime.zone.now
NoMethodError: undefined method `zone' for DateTime:Class
>> DateTime.now
=> Sun, 16 Aug 2015 00:37:50 -0400
>> Time.zone.now.to_datetime
=> Sun, 16 Aug 2015 04:37:48 +0000
```

That last one is what I think it should suggest instead.

**Comment by maxjacobson** [comment]
(The cop also suggests things like `DateTime.zone.parse` which also doesn't exist)

**Comment by maxjacobson** [comment]
You know, I think this might be resolved in 0.33.0. Now it recommends `Time.zone.now` instead of `DateTime.zone.now`. That's close enough, I think.

**Comment by palkan** [comment]
This has been [resolved](https://github.com/bbatsov/rubocop/pull/1995) in 0.33.0.
